docs/create-pipeline-all-amps-images.md

```diff
@@ -0,0 +1,11 @@
+# ACS Pipeline-all-amps Repo & Share Docker Image
```

Comment:
Delete this. The same text is in the README

Reply:
Done